.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
```

Comment:
I think trailing is at the end of the line.
None the less, if the rule is there, it's because it's a good practice, and if xcode doesn't do it, then we could do it. I don't think it's hard work to do it. But I agree I don't see a great benefit from it.